sw/device/silicon_creator/lib/dice.h

```diff
@@ -107,4 +116,23 @@ rom_error_t dice_cdi_1_cert_build(manuf_certgen_inputs_t *inputs,
                                   attestation_public_key_t *cdi_1_pubkey,
                                   uint8_t *cert, size_t *cert_size);
 
-#endif  // OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DICE_H_
+OT_WARN_UNUSED_RESULT
+rom_error_t tpm_attestation_keygen(tpm_key_t desired_key,
+                                   hmac_digest_t *pubkey_id,
+                                   attestation_public_key_t *pubkey);
+
```

Comment:
These are all missing docstrings.

Reply:
added

Reply:
Hmm I don't see the docstrings, did you forget to upload them in your latest push?

Reply:
yes, have not saved the latest edit, fixed.